pkg/kubectl/cmd/attach.go

```diff
@@ -46,7 +48,11 @@ var (
 
 		# Switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 123456-7890
 		# and sends stdout/stderr from 'bash' back to the client
-		kubectl attach 123456-7890 -c ruby-container -i -t`)
+		kubectl attach 123456-7890 -c ruby-container -i -t
+
+		# Get output from the first pod of a Replication Controller named nginx
+		kubectl attach rc/nginx
```

Comment:
Can you add an example using something else other than a replication controller? We want people to start using ReplicaSets in favor of ReplicationControllers.